operations/node_management/Reboot_NCNs.md

```diff
@@ -137,6 +137,18 @@ The `kubectl` command is installed.
         This check will need to be run after all worker node have been rebooted. Ensure that the checks have been run to check BGP peering sessions on the spine switches \(instructions will vary for Aruba and Mellanox switches\)
 
         If there are BGP Peering sessions that are not ESTABLISHED on either switch, refer to [Check BGP Status and Reset Sessions](../network/metallb_bgp/Check_BGP_Status_and_Reset_Sessions.md).
+        
+    1. Check for components that are in an failed state in CFS.
+    
+        Components can be found by running the following command:
+        ```bash
+         ncn-m001# cray cfs components list --status failed
```

Comment:
The beginning of the command prompt (the letter n) should be indented to the same depth as the three backticks that surround it